src/amber.cr

```diff
@@ -34,16 +21,37 @@ module Amber
       instance
     end
 
+    def self.key_generator
+      instance.key_generator
+    end
+
```

Comment:
Yest we should create a config class that we can set a hash of settings something more manageable